doc/lua_api.md

```diff
@@ -2191,6 +2191,75 @@ Table of resulting tool uses:
 
 
 
+Wear Bar Color
+==============
+
+'Wear Bar' is a property of items that defines the coloring
+of the bar that appears under damaged tools.
+If it is absent, the old behavior of green, yellow, red is
+used.
+
+Wear bar colors definition
+--------------------------
+
+### Syntax
+
+For non-blending mode:
+```lua
+{
+    {
+        color="#ff00ff",
+        min_durability=0.2, -- minimum durability is inclusive
+        max_durability=0.3  -- maximum durability is exclusive
+    },
+    {
+        color="#c0ffee",
+        min_durability=0.45,
+        max_durability=0.6
+    },
+    default="#ffff00",      -- color to use if no other ranges match the durability
+    blend=false
+}
+```
+
+For blending mode:
+```lua
+{
+    [0.2] = "#ff00ff",      -- specify color for a specific durability percent, and blend between them
+    [0.45] = "#c0ffee",
+    default="#ffff00",      -- used for 0.0% and 100.0% durability if no color explicitly specified
+    blend=true
+}
+```
+
+### Blend Mode `blend`
+
+When true, blends smoothly between each defined color point.
+When false, no interpolation is used, and instead colors are defined for ranges of durabilities.
+`blend` is optional and defaults false.
+
+### Default Color `default`
+
+When `blend` is false, the color to use if the percentage of remaining durability does not match any values.
+When `blend` is true, the color to use on the 'outside' of the range of colors, if no such color is defined
+
+### Color values
+
+In blending mode, specified as `float` keys assigned to a `ColorString` values.
+In non-blending mode, specified as tables containing the following keys:
+* `min_durability`: `float`, minimum durability to show color at (inclusive)
+* `max_durability`: `float`, maximum durability to show color at (exclusive)
+* `color`:          `ColorString`, color to use for bar in the specified range
```

Comment:
There is no need to align for this value (with lines above) as it will be shown the same as a Markdown file.
```suggestion
* `color`: `ColorString`, color to use for bar in the specified range
```